src/applications/mhv/secure-messaging/containers/AuthorizedRoutes.jsx

```diff
@@ -13,7 +13,7 @@ import SearchResults from './SearchResults';
 
 const AuthorizedRoutes = () => {
   return (
-    <div className="vads-u-flex--fill">
+    <div className="secure-messaging vads-u-flex--fill">

```

Comment:
Let's double check if this is really needed here. Maybe I missed, but I don't see anywhere where we would be relying on this class. Additionally, we are already declaring a `secure-messaging-container` class in App.jsx. However, if there is a plan for this class to be used, we can keep it

Reply:
We're not using this classname officially for styling yet but it did help with my debugging by identifying the sections. For that purpose I'll change it to a data-testid. Thanks!